app/models/foreman_openscap/policy.rb

```diff
@@ -168,8 +170,37 @@ def to_enc
       }.merge(period_enc)
     end
 
+    def should_validate?(step_name)
+      if new_record? && wizard_initiated?
+        step_index > step_to_i(step_name)
+      elsif new_record? && !wizard_initiated?
+        true
+      else
+        persisted?
+      end
+    end
+
+    def wizard_initiated?
+      @wizard_initiated
+    end
+
+    def update_attrs
```

Comment:
`update_attrs` strikes me as too general name (e.g., I expected to find more attributes to update here) where this update deals only with `period`. Would you consider renaming this method?